app/controllers/network_services_controller_test.go

```diff
@@ -0,0 +1,519 @@
+package controllers
+
+import (
+	"fmt"
```

Comment:
Group std imports together

Reply:
`go fmt` is sorting them that way (and there's a check at Makefile I
want to obey ;)) .

Reply:
`gofmt` doesn't account for how imports are grouped so `gofmt` still passes if you group stdlib separately, example: https://github.com/cloudnativelabs/kube-router/blob/master/app/controllers/network_routes_controller.go#L4-L15. The grouping is just good practice since it's easier to find packages that way.

Reply:
Gotcha, done (588b6429).